frontend/TestRun/ResourcesInfo.svelte

```diff
@@ -350,6 +370,7 @@
         {#each sortResourcesByKey(resources, sortHeader, sortAscending) as resource (resource.name)}
             <tr class:d-none={filterResource(resource)}>
                 <td>{resource.instance_info.provider}</td>
+                <td>{resource.instance_info.instance_type}</td>
```

Comment:
this will be None for existing tests (sent with old client) - how will it look? shouldn't we fallback to `N/A`?

Reply:
Fixed